src/test_l1_dsHdmiIn.c

```diff
@@ -155,7 +155,11 @@ void test_l1_dsHdmiIn_negative_dsHdmiInInit(void) {
     UT_ASSERT_EQUAL(dsHdmiInInit(), dsERR_NONE);
 
     // Step 2: Call dsHdmiInInit() again without terminating the HDMI input sub-system
-    DS_ASSERT_AUTO_TERM_NUMERICAL(dsHdmiInInit(), dsERR_ALREADY_INITIALIZED);
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
https://github.com/rdkcentral/ut-core/discussions/55 -> please update to support the KVP module , so we can commit this code as a valid upgrade, that can change configuration at runtime.